tools/testbench/topology_ipc4.c

```diff
@@ -359,6 +434,16 @@ int tb_set_up_widget_base_config(struct testbench_prm *tp, struct tplg_comp_info
 	/* copy the basecfg into the ipc payload */
 	memcpy(comp_info->ipc_payload, &comp_info->basecfg, sizeof(struct ipc4_base_module_cfg));
 
+	/* Copy ext config for process type */
+	if (comp_info->module_id == TB_PROCESS_MODULE_ID) {
+		base_cfg = comp_info->ipc_payload;
+		base_cfg_ext = comp_info->ipc_payload + sizeof(*base_cfg);
+		base_cfg_ext->nb_input_pins = pins->num_input_pins;
+		base_cfg_ext->nb_output_pins = pins->num_output_pins;
+		tb_set_pin_formats(comp_info, base_cfg, base_cfg_ext, 0);
+		tb_set_pin_formats(comp_info, base_cfg, base_cfg_ext, 1);
```

Comment:
Here readable enum/defines for 0/1 would help to understand what this code does easier.